examples/sdk-examples/src/examples/attributeFilter/AttributeParentChildFilterButtonWithPlaceholderExample.tsx

```diff
@@ -18,10 +16,11 @@ import {
     newPositiveAttributeFilter,
 } from "@gooddata/sdk-model";
 import { Ldm, LdmExt } from "../../ldm";
+import { PlaceholdersProvider } from "@gooddata/sdk-ui";
 
 const stateFilterPlaceholder = newPlaceholder<IAttributeFilter>(
     newPositiveAttributeFilter(attributeDisplayFormRef(Ldm.LocationState), {
-        uris: [],
+        uris: ["/gdc/md/xms7ga4tf3g3nzucd8380o2bev8oeknp/obj/2210/elements?id=6340116"],
```

Comment:
Should we prefer `newNegativeFilter` with empty array? So that everything is selected by default? The URI might be confusing to first time users (where did we get it etc.).